.gitignore

```diff
@@ -2,4 +2,5 @@ Mentoring QT project.codeworkspace.code-workspace
 /.vscode
 /build-BasicGUI-Desktop_Qt_5_15_2_MinGW_64_bit-Debug
 /build
-/CMakeLists.txt.user
\ No newline at end of file
+CMakeLists.txt.user
+/BasicGUI/CMakeLists.txt.user
```

Comment:
Patterns for CMakeLists.txt.user could be replace with one `*.txt.user`

Reply:
Resolved